unicode-catalog/src/main/kotlin/com/avail/tools/unicode/Catalog.kt

```diff
@@ -368,7 +374,7 @@ internal class Catalog
 
 	/**
 	 * Answer a [JSON-friendly representative][JSONFriendly] of the
-	 * [complete set of non-ASCII,][allSymbolicCodePoints].
+	 * [complete&#32;set&#32;of&#32;non-ASCII,][allSymbolicCodePoints].
```

Comment:
...`of the [set][allSymbolicCodePoints] of code points that are neither ASCII nor alpha-numeric.`